src/com/projectkorra/projectkorra/waterbending/multiabilities/WaterArmsWhip.java

```diff
@@ -599,11 +584,11 @@ public void setWhipSpeed(final int whipSpeed) {
 	}
 
 	public long getHoldTime() {

```

Comment:
This method should be renamed to better reflect the specific variable it accesses. Something like `getGrabDuration` would be more fitting.